shared/queues.py

```diff
@@ -26,7 +25,7 @@ class Queue:
     with qsize(), since your single-threaded uasyncio application won't be
     interrupted between calling qsize() and doing an operation on the Queue.
     """
-    _attempt_delay = 0.01
+    _attempt_delay = 10         # milliseconds
```

Comment:
Curious about this change in delay being 1000x its prior value.

Reply:
it now uses `sleep_ms()` rather than `sleep()` so 0.010 seconds == 10 ms.